eppo_core/src/event_ingestion/event_delivery.rs

```diff
@@ -1,9 +1,9 @@
-use std::collections::HashSet;
-use crate::events::event::Event;
+use crate::event_ingestion::event::Event;
```

Comment:
nit: I usually split imports into 3 sections in this order:
- std
- external
- local

Reply:
i just let the IDE handle imports for me 😂